### ISSUE TITLE
You are able to press buttons (and therefore run commands) on computer that has been locked

### ISSUE THREAD

**Issue by celgit** [timothycrosley/streamdeck-ui#282]
My environment:
PopOs 22.04
Installed 2022-12-11, followed this guide here after that.
Works fine.
I use a regular Elgato Stream Deck (v1).

A couple of the buttons have been configured to run a bash-script which in turn runs xrandr-commands to toggle monitors off and off (or outputs to be precise).

What I noticed is that even if my computer is locked, I can still press the buttons on the stream deck and therefore run xrandr-commands without unlocking the computer.
(The way I noticed it was that when I wake the computer from sleep, it seems to default to enable all connected monitors, but one of them is used for another thing at the time and I can't see the login prompt. I then press the "toggle monitor x"-button and it disables it and the login-prompt is shown on another screen.)

tl;dr: 
You can still run commands with the stream deck on a locked linux computer with the streamdeck-ui-solution.

Offtopic:
Thanks for this app or whatever you may call it, works like a charm (too good if you consider the ticket here ;)), really appreciate the opportunity to use my stream deck in linux! <3

**Comment by NicolasGoeddel** [comment]
Oh yes, I also found out that I can start Zoom calls, activate my camera and in general start applications using my Streamdeck while Gnome is showing me its lockscreen. It would be very cool if there would be an option to also lock the stream deck when the screen is locked.

**Comment by ianfloats** [comment]
Having this be optional per button would be ideal.  Some buttons are not sensitive at all, like volume control.

Having the device show a lock icon for buttons that are currently inaccessible would be extra slick.

**Comment by StormyIceLeopard** [comment]
This is a security issue that needs to be addressed. Pull Request https://github.com/timothycrosley/streamdeck-ui/pull/235 should be verified and merged and released asap. @dodgyrabbit

**Comment by 4s3ti** [comment]
hi, this project seems to be stale and a new fork as been made with the intention of replacing this one.

if this still an issue please reopen it at: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui
